pkg/compression/compression.go

```diff
@@ -13,6 +13,17 @@ import (
 	"github.com/ulikunitz/xz"
 )
 
+const (
+	// GzipCompression indicates gzip compression.
+	GzipCompression = "gzip"
```

Comment:
Now that the constants exist, could the hard-coded default in `copy.Image` use the symbolic constant instead of a string?